worker/worker.py

```diff
@@ -0,0 +1,122 @@
+"""
+Stencila Hub Worker.
+
+A Celery app for running jobs.
+See https://docs.celeryproject.org/en/latest/userguide/tasks.html.
+"""
+
+from contextlib import contextmanager, redirect_stdout, redirect_stderr
+from datetime import datetime
+import io
+import json
+import os
+import sys
+import time
+
+from celery import Celery
+from celery.exceptions import SoftTimeLimitExceeded
+
+from execute.process import ProcessSession
+
+
+# Setup the Celery app
+celery = Celery("worker", broker=os.environ["BROKER_URL"], backend='rpc://')
+celery.conf.update(
+    # By default Celery will keep on trying to connect to the broker forever
+    # This overrides that. Initially try again immediately, then add 0.5 seconds for each
+    # subsequent try (with a maximum of 3 seconds).
+    # See https://github.com/celery/celery/issues/4296
+    broker_transport_options={
```

Comment:
This seems like it's the same as `jobs.py` above, can these constants be shared somewhere?